Source/Core/Core/HW/EXI/BBA/TAP_Win32.cpp

```diff
@@ -66,11 +67,16 @@ bool IsTAPDevice(const TCHAR* guid)
 
         if (status == ERROR_SUCCESS && data_type == REG_SZ)
         {
-          if (!_tcscmp(component_id, TAP_COMPONENT_ID) && !_tcscmp(net_cfg_instance_id, guid))
+          component_id_sub = _tcsstr(component_id, TAP_COMPONENT_ID);
+
+          if(component_id_sub)
```

Comment:
Missing space before the `(`, did you run clang-format? https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#checking-and-fixing-formatting-issues

Reply:
Note that if you use Visual Studio, you can press Control+K followed by Control+D to auto-format the current document using the clang-format rules (without needing to install clang-format separately).

Reply:
I use vim, but let me fix this real quick.